tools/testbench/topology.c

```diff
@@ -64,8 +64,11 @@ void register_comp(int comp_type)
 
 	/* get index of comp in shared library table */
 	index = get_index_by_type(comp_type, lib_table);
-	if (index < 0)
-		return;
+	if (index < 0 && comp_ext) {
```

Comment:
Instead of "index < 0" check here "comp_type == SOF_COMP_NONE"

Reply:
oh yes, thanks!